src/components/Context/CartContext.tsx

```diff
@@ -0,0 +1,100 @@
+import { PropsWithChildren, createContext, useState } from "react";
+import { FeaturedProductObject } from "../App";
+
+export interface ItemToPurchase {
+  quantity: number;
+  product: FeaturedProductObject;
+}
+
+export const CartContext = createContext<ItemToPurchase[]>([]);
```

Comment:
Thank you!  This is working great!  It seems to work just as well without the export before interface CartContextType.  I think that's because I'm only using that interface here on the CartContext.tsx component.  Do you still recommend having it exported?  

I'm going to implement this for ProductContext, too!

Reply:
<img width="591" alt="Screenshot 2024-03-02 at 6 58 00 PM" src="https://github.com/Zevgon/challenge-review/assets/43486273/12dda13d-c9a1-4c29-a8c4-c0c48c791c53">

Did this image work?  I'm trying to send a screenshot.  I'm confused about how to initialize findProduct() inside ProductsContext.  I have typed it as a function that returns a Product but when initializing it, I can't seem to get it to work unless I use @ts-ignore.  Nothing is breaking but I'd like to initialize it correctly